app/src/main/java/com/example/geoquiz/MainActivity.kt

```diff
@@ -22,18 +22,38 @@ class MainActivity : AppCompatActivity() {
         Question(R.string.question_asia, true)
     )
     private var currentIndex = 0
+    private val tag = "MainActivity"
```

Comment:
Can be made `const`. In my project at least, we prefer to name constants in capital leters i.e TAG in this case